Common/Systems/ModPlayers/ClassModPlayer.cs

```diff
@@ -24,24 +24,18 @@ public override void ModifyStartingInventory(IReadOnlyDictionary<string, List<It
 
 		vanillaItems.Clear();
 
-		// We want the first slot to be empty so the chosen class' weapon can
-		// be placed there.
-		var item = new Item();
-		item.TurnToAir();
-		vanillaItems.Add(item);
-
-		item = new Item(ItemID.WoodenSword);
+		var item = new Item(ItemID.WoodenSword);
 		PoTInstanceItemData data = item.GetInstanceData();
-		data.Rarity = Enums.ItemRarity.Magic;
+		data.Rarity = ItemRarity.Magic;
 		PoTItemHelper.Roll(item, PoTItemHelper.PickItemLevel());
 		vanillaItems.Add(item);
 		
-		item = new Item(ItemID.CopperPickaxe);
-		vanillaItems.Add(item);
+		vanillaItems.Add(new Item(ItemID.CopperPickaxe));
```

Comment:
Is there a reason we're adding to the vanilla list? It's meant to be "by-mod", and I don't see a reason to avoid that.

Reply:
Idk, was just following the path laid before me